doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,820 @@
+    * The logging format in the standard logging mode to align it with the format used in the multi-domain logging mode.
+    * Support for alternative configurations to use the :ref:`file suffix feature from Zephyr <app_build_file_suffixes>`.
+      The following file suffixes are supported as alternative configurations:
+
+      * ``flash_and_run``
+      * ``heap_memory``
+      * ``static_memory``
+      * ``multiple_threads``
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+|no_changes_yet_note|
+
+Wi-Fi drivers
+-------------
+
+* Added the :ref:`nrf70_wifi_tx_power_calculation` section to the :ref:`nrf70_wifi` page.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Binary libraries
+----------------
+
+* :ref:`liblwm2m_carrier_readme` library:
+
+  * Updated the :ref:`req_appln_limitations` page to clarify carrier-specific requirements.
+    Added overlay files and documentation to :ref:`serial_lte_modem` application and :ref:`lwm2m_carrier` sample to guide in the correct usage of LwM2M carrier library for SoftBank and LG U+.
+
+Bluetooth libraries and services
+--------------------------------
+
+* :ref:`hogp_readme` library:
+
+  * Updated the :c:func:`bt_hogp_rep_read` function to forward the GATT read error code through the registered user callback.
+    This ensures that API user is aware of the error.
+
+Common Application Framework
+----------------------------
+
+|no_changes_yet_note|
+
+Debug libraries
+---------------
+
+|no_changes_yet_note|
+
+DFU libraries
+-------------
+
+|no_changes_yet_note|
+
+Gazell libraries
+----------------
+
+|no_changes_yet_note|
+
+Security libraries
+------------------
+
+|no_changes_yet_note|
+
+Modem libraries
+---------------
+
+|no_changes_yet_note|
+
+Multiprotocol Service Layer libraries
+-------------------------------------
+
+|no_changes_yet_note|
+
+Libraries for networking
+------------------------
+
+|no_changes_yet_note|
+
+Libraries for NFC
+-----------------
+
+|no_changes_yet_note|
+
+nRF RPC libraries
+-----------------
+
+* Added the :ref:`nrf_rpc_dev_info` library for obtaining information about a device connected through the :ref:`nrfxlib:nrf_rpc`.
+
+Other libraries
+---------------
+
+|no_changes_yet_note|
+
+Security libraries
+------------------
+
+|no_changes_yet_note|
+
+Shell libraries
+---------------
+
+|no_changes_yet_note|
+
+Libraries for Zigbee
+--------------------
+
+|no_changes_yet_note|
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* Added semantic version support to :ref:`nrf_desktop_config_channel_script` Python script for devices that use the SUIT DFU.
```

Comment:
This changelog entry is already part of the NCS 2.8. Please remove the following

```suggestion
```